src/DigitalProductPassport/MetalArtifact/DataSheet_v0.1.py

```diff
@@ -0,0 +1,102 @@
+from typing import List, Optional
+
+from definition_tooling.converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class EnStandardCertification(CamelCaseModel):
+    en_standard_code: Optional[str] = Field(
+        None,
+        title="EN Standard Code",
+        max_length=20,
+        description="The identification code of the EN standard that the product complies with",
```

Comment:
Missmatch with implemented and what's in the spreadsheet
```suggestion
        description="The identification code of the EN standard that the product is compliant with",
```